apps/test-suite/tests/MediaLibrary.js

```diff
@@ -94,18 +96,46 @@ export async function test(t) {
     let album;
     let files;
 
-    t.beforeAll(async () => {
+    async function initializeAsync() {
       await Permissions.askAsync(Permissions.CAMERA_ROLL);
       files = await getFiles();
       testAssets = await getAssets(files);
       album = await MediaLibrary.getAlbumAsync(ALBUM_NAME);
-      if (album == null) album = await createAlbum(testAssets, ALBUM_NAME);
-      else await MediaLibrary.addAssetsToAlbumAsync(testAssets, album, true);
+      if (album == null) {
+        album = await createAlbum(testAssets, ALBUM_NAME);
+      } else {
+        await MediaLibrary.addAssetsToAlbumAsync(testAssets, album, true);
+      }
+    }
+
+    async function cleanupAsync() {
+      await MediaLibrary.deleteAssetsAsync(testAssets);
+      await MediaLibrary.deleteAlbumsAsync(album);
+    }
+
+    t.beforeAll(async () => {
+      // NOTE(2020-06-03): The `initializeAsync` function is flaky on Android; often the
+      // `addAssetsToAlbumAsync` method call inside of `createAlbum` will fail with the error
+      // "Could not get all of the requested assets". Usually retrying a few times works, so we do
```

Comment:
Just created an issue for this: https://github.com/expo/expo/issues/8653
I've assigned myself as I'd like to revisit some modules that I'm an owner of.